### PR TITLE
Add New|Get-CsCloudCallDataConnection cmdlets public docs

### PR DESCRIPTION
**Pull request by 2012ucp1544**
This change contains:

Add Get-CsCloudCallDataConnection docs page
Add New-CsCloudCallDataConnection docs page
Update TOC

### CODE REVIEW

**Comment by yogkumgit** [comment]
@get-itips @dariomws @tseward

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by pbafna03 — APPROVED** [review]

File: teams/teams-ps/teams/New-CsCloudCallDataConnection.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+00000000-0000-0000-0000-000000000000
+```
+
+Returns a token value, which is needed when configuring your on-premises environment with Set-CsCloudCallDataConnector.
+
+
+## PARAMETERS
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (get-itips) [inline teams/teams-ps/teams/New-CsCloudCallDataConnection.md:47]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: teams/teams-ps/teams/Get-CsCloudCallDataConnection.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+-----
+00000000-0000-0000-0000-000000000000
+```
+
+Returns a token value, which is needed when configuring your on-premises environment with Set-CsCloudCallDataConnector.
+
+## PARAMETERS
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (get-itips) [inline teams/teams-ps/teams/Get-CsCloudCallDataConnection.md:45]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by dariomws — APPROVED** [review]
I'm ok after suggestions.

File: teams/teams-ps/teams/Get-CsCloudCallDataConnection.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+### None
+
+## OUTPUTS
+
+### System.Object
+
+## NOTES
+
+Get-CsCloudCallDataConnection cmdlet is now supported only from Teams PowerShell Module versions 4.6.0 or later.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsCloudCallDataConnection.md:57]:
```suggestion
The Get-CsCloudCallDataConnection cmdlet is only supported from Teams PowerShell Module versions 4.6.0 or later.
```

File: teams/teams-ps/teams/New-CsCloudCallDataConnection.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+This cmdlet creates an online call data connection.
+
+## SYNTAX
+
+```powershell
+New-CsCloudCallDataConnection
+```
+
+## DESCRIPTION
+This cmdlet creates an online call data connection. If you get an error that the connection already exists, this means that the call data connection already exists for your tenant. In this case, run Get-CsCloudCallDataConnection.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsCloudCallDataConnection.md:28]:
```suggestion
This cmdlet creates an online call data connection. If you get an error that the connection already exists, it means that the call data connection already exists for your tenant. In this case, run Get-CsCloudCallDataConnection.
```

File: teams/teams-ps/teams/New-CsCloudCallDataConnection.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+### None
+
+## OUTPUTS
+
+### System.Object
+
+## NOTES
+
+New-CsCloudCallDataConnection cmdlet is now supported only from Teams PowerShell Module versions 4.6.0 or later.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsCloudCallDataConnection.md:59]:
```suggestion
The New-CsCloudCallDataConnection cmdlet is only supported from Teams PowerShell Module versions 4.6.0 or later.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @2012ucp1544, kindly commit the suggested changes. Many thanks.

File: teams/teams-ps/teams/New-CsCloudCallDataConnection.md (hunk 1 above)
Comment (2012ucp1544) [reply teams/teams-ps/teams/New-CsCloudCallDataConnection.md]: Incorporated review comments, please review again.

File: teams/teams-ps/teams/Get-CsCloudCallDataConnection.md (hunk 2 above)
Comment (2012ucp1544) [reply teams/teams-ps/teams/Get-CsCloudCallDataConnection.md]: Incorporated review comments, please review again.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**